components/DashboardProfile/index.js

```diff
@@ -9,11 +9,18 @@ import { Button } from 'react-bootstrap';
 class DashboardProfile extends React.Component {
   constructor(props) {
     super(props);
+    this.messages = {
+      emailValidation: 'Please enter a valid email address or leave empty',
+      profileSaved: 'Profile saved!',
+      enterNickname: 'Please edit your profile to set a nickname'
+    };
     this.state = {
+      disabled: false,
       edit: false,
       user_metadata: {
```

Comment:
why not opt for `this.state.profile` just to keep things separate. 

React code: `state.profile` `props.profile`. Auth0 code: `user_metadata`

Reply:
check